docs/roles/integrator/integrating.md

```diff
@@ -0,0 +1,372 @@
+---
+id: integrating
+title: Integrating with NEAR
+sidebar_label: Integrating with NEAR
+---
+
+## Overview
+
+Integrating with NEAR is an opportunity for wallets, exchanges and oracles to take advantage of NEAR's scalability and performance for the benefit of their users.
+
+### Prerequisites
+
+- Hardware: Machine requirements are [here](https://docs-pr-136.onrender.com/docs/roles/validator/hardware)
+- Development:  Development environment requirements are available [here](https://docs.nearprotocol.com/docs/local-setup/local-dev-testnet)
+- Regulations: All regulations that apply to your jurisdiction are your sole responsibility including KYC / AML / CFT / BSA (Know Your Customer (KYC), Anti-Money Laundering (AML), Combating the Financing of Terrorism (CFT), Bank Secrecy Act (BSA))
+
+
+<blockquote class="info">
+<strong>did you know?</strong><br><br>
+
+The Near team recommends that custody wallets and exchanges track all shards to remove any possibility of invalid state transition. `tracking_shards` is a node configuration parameter available in the `~/.near/config.json` file which supports tracking all shards. You can also run 8 separate nodes that track each shard individually and route requests to the right nodes for higher performance.
+
+</blockquote>
+
+
+### Server Setup
+
+Please see [hardware requirements](https://docs-pr-136.onrender.com/docs/roles/validator/hardware) and details of how to [run a NEAR node](https://docs-pr-136.onrender.com/docs/local-setup/running-testnet).
+
+## Blockchain
+
+### Finality
+
+To learn whether a block is final or note, NEAR accepts a flag in calls to all RPC endpoints that indicates whether the requester wants the response as of the last block or the last *final* block.
+
+Exchanges should only use the last final block.
+
+<blockquote class="warning">
+<strong>work in progress</strong> <span>this feature is currently under development</span><br><br>
+
+Currently the list of parameters in the example below *does not include* the finality flag mentioned above.
+
+</blockquote>
+
+```sh
+# query format: account/<account_id>
+http post http://rpc.nearprotocol.com jsonrpc=2.0 method=query  \
+                                      params:="[\"account/test.near\",\"\"]" \
```

Comment:
you can use single quotes to make this simpler:
`params:='["account/test.near", ""]'`

Reply:
nice. will update

Reply:
✅